src/components/Person/Person.jsx

```diff
@@ -1 +1,44 @@
-// export const Person = ({ person }) => ();
+import { bool, number, string } from 'prop-types';
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const isGender = sex === 'f'
```

Comment:
why do you use `is` prefix, is it boolean?

Reply:
Still not fixed